src/App.test.js

```diff
@@ -1,9 +1,16 @@
 import { render, screen } from '@testing-library/react';
+import renderer from 'react-test-renderer';
 import App from './App';
 
 test('renders learn react link', () => {
   render(<App />);
   const linkElement = screen.getByText(/Magicians/i);
-
   expect(linkElement).toBeInTheDocument();
 });
+
+describe('App', () => {
+  it('renders correctly', () => {
+    const tree = renderer.create(<App />);
+    expect(tree).toMatchSnapshot();
+  });
```

Comment:
- [ ] Well done on adding another test. However, please consider the following project requirements. 🙏

1. [Write unit tests for the files operate.js and calculate.js using Jest](https://github.com/microverseinc/curriculum-react-redux/blob/main/math-magicians/project_tests.md#:~:text=Write%20unit%20tests%20for%20the%20files%20operate.js%20and%20calculate.js%20using%20Jest.).
2. [Use Jest snapshots to test the components](https://github.com/microverseinc/curriculum-react-redux/blob/main/math-magicians/project_tests.md#:~:text=Use%20Jest%20snapshots%20to%20test%20the%20components). Please add snapshot tests for the other components.
3. [Use React Testing Library to simulate user interaction](https://github.com/microverseinc/curriculum-react-redux/blob/main/math-magicians/project_tests.md#:~:text=Use%20React%20Testing%20Library%20to%20simulate%20user%20interaction).